pkg/helpers/helpers_test.go

```diff
@@ -174,12 +120,12 @@ EPDesL0rH+3s1CKpgkhYdbJ675GFoGoq+X21QaqsdvoXmmuJF9qq9Tq+JaWloUNq
 	pemBuffer := bytes.Buffer{}
 	pem.Encode(&pemBuffer, pemBlock)
 
-	if pemBuffer.String() != expectedPrivateKeyString {
-		t.Fatalf("Private Key did not match expected format/value")
+	if !strings.HasPrefix(pemBuffer.String(), "-----BEGIN RSA PRIVATE KEY-----") {
```

Comment:
Can we add lines that test the following for the Private Key?

1. privateKey.N.BitLen() equals the SSHKey bit size (which is set at 4096 for us) 
2. privatKey.Validate() does not throw an err

Reply:
Good call--I added that.